submissions/hamzamogni/main.py

```diff
@@ -0,0 +1,122 @@
+import os
+import multiprocessing as mp
```

Comment:
Only std libs are allowed

Reply:
those are part of python's std lilb

https://docs.python.org/3/library/multiprocessing.html
https://docs.python.org/3/library/os.html#module-os


Thanks

Reply:
I am using os, multiprocessing, mmap, collections and threading ... all part of standard library

Reply:
mmap: https://docs.python.org/3/library/mmap.html#module-mmap
threading: https://docs.python.org/3/library/threading.html#module-threading
collections: https://docs.python.org/3/library/collections.html#module-collections